src/main/java/io/stargate/sgv2/jsonapi/exception/JsonApiException.java

```diff
@@ -63,7 +65,12 @@ public CommandResult get() {
   public CommandResult.Error getCommandResultError(String message, Response.Status status) {
     Map<String, Object> fieldsForMetricsTag =
         Map.of("errorCode", errorCode.name(), "exceptionClass", this.getClass().getSimpleName());
-    SmallRyeConfig config = ConfigProvider.getConfig().unwrap(SmallRyeConfig.class);
+    SmallRyeConfig config;
```

Comment:
Can you add comment on why this is done. Can the `config` made static

Reply:
Done. Fixed in this commit [02bf4d0](https://github.com/stargate/data-api/pull/905/commits/02bf4d024b669e0e7ae70f09c804b2157646b501).

Reply:
@maheshrajamani reverted this commit, since it breaks the JsonApiException Initialization during application startup.